apis/core/hydra/job.ttl

```diff
@@ -41,6 +44,8 @@ cc:JobCollection
         "Start transformation" ;
       hydra:method
         "POST" ;
+      hydra:expects
+        <shape/job/trigger> ;
```

Comment:
This file is missing `<shape/job/trigger> a sh:Shape .`

Reply:
good catch 🏉